clustered/server/src/main/java/org/ehcache/clustered/server/store/ClusterTierActiveEntity.java

```diff
@@ -113,6 +116,9 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(ClusterTierActiveEntity.class);
   static final String SYNC_DATA_SIZE_PROP = "ehcache.sync.data.size.threshold";
   private static final long DEFAULT_SYNC_DATA_SIZE_THRESHOLD = 4 * 1024 * 1024;
```

Comment:
Why are these properties  java properties instead of normal Ehcache configuration ones?

Reply:
I am guessing we do not want users to play around with these. However keeping it as system properties will give us an opportunity to tune/study the performance. That is my best guess.

Reply:
Why can't we study performance with a clustered configuration?